xyz.aguno.CubeTimer.json

```diff
@@ -0,0 +1,38 @@
+{
+    "app-id" : "xyz.aguno.CubeTimer",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "cubetimer",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "cubetimer",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/herpiko/cubetimer.git+a04f8eed59bbda28bb82a60b05288e85a50d6a8b"
```

Comment:
`tag: ...` + `commit: ...` or at least `commit: ...`

Reply:
this is not the syntax at all.